includes/js/functions.js

```diff
@@ -182,7 +182,7 @@ function prepareExchangedData(data, type, key, fileName = '', functionName = '')
             }
         }
     } else if (type === 'encode') {
-        if (parseInt($('#encryptClientServer').val()) === 0) {
+        if (parseInt($('#encryptClientServerStatus').val()) === 0) {
```

Comment:
Well seen, I wanted to do that without success but I forgot to modify this line!